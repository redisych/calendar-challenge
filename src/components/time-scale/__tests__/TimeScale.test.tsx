import * as React from 'react';
import * as enzyme from 'enzyme';
import { TimeScale } from '..';
import toJson from 'enzyme-to-json';

const getWrapper = (): any => {
    return enzyme.shallow(<TimeScale />);
};

describe('TimeScale', () => {
    it('should be defined', () => {
        const wrapper = getWrapper();

        expect(wrapper).toBeDefined();
    });

    it('should match snapshot', () => {
        const wrapper = getWrapper();

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
