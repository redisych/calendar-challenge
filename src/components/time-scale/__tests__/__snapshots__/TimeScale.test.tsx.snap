// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimeScale should match snapshot 1`] = `
<div
  className="time-scale"
>
  <Tick
    key="0"
    minute={0}
  />
  <Tick
    key="30"
    minute={30}
  />
  <Tick
    key="60"
    minute={60}
  />
  <Tick
    key="90"
    minute={90}
  />
  <Tick
    key="120"
    minute={120}
  />
  <Tick
    key="150"
    minute={150}
  />
  <Tick
    key="180"
    minute={180}
  />
  <Tick
    key="210"
    minute={210}
  />
  <Tick
    key="240"
    minute={240}
  />
  <Tick
    key="270"
    minute={270}
  />
  <Tick
    key="300"
    minute={300}
  />
  <Tick
    key="330"
    minute={330}
  />
  <Tick
    key="360"
    minute={360}
  />
  <Tick
    key="390"
    minute={390}
  />
  <Tick
    key="420"
    minute={420}
  />
  <Tick
    key="450"
    minute={450}
  />
  <Tick
    key="480"
    minute={480}
  />
  <Tick
    key="510"
    minute={510}
  />
  <Tick
    key="540"
    minute={540}
  />
  <Tick
    key="570"
    minute={570}
  />
  <Tick
    key="600"
    minute={600}
  />
  <Tick
    key="630"
    minute={630}
  />
  <Tick
    key="660"
    minute={660}
  />
  <Tick
    key="690"
    minute={690}
  />
  <Tick
    key="720"
    minute={720}
  />
</div>
`;
